myapp/db/migrate/20221223073346_add_index_to_tasks_status.rb

```diff
@@ -0,0 +1,9 @@
+class AddIndexToTasksStatus < ActiveRecord::Migration[6.0]
+  def up
+    add_index :tasks, :status
```

Comment:
As I use key word search here (`LIKE %xxxx%`) and referred to this [stackoverflow](https://stackoverflow.com/a/1395881), I see there is no better index for keyword search. So I only implemented the index for status. If we wanna do keyword search for this app, it's suggested to use other tools like Elastic Search or so that's better at this job.